src/main/webapp/src/app/location/location.component.ts

```diff
@@ -50,6 +51,7 @@ export class LocationComponent{
   }
 
   enableNavigatorLocation(): void {
+    this.isLocationButtonClicked = true;
```

Comment:
Does anything ever set this back to false? or is the component just unloaded?
Could you add a comment explaining this a bit?

Reply:
When the user clicks on the button to retrieve their location and the location has been fetched, it redirects the user to the '/home' page; so the spinner goes away. Is it ok to keep it like that?